yt/visualization/volume_rendering/tests/test_points.py

```diff
@@ -66,4 +66,3 @@ def test_points_vr(self):
         sc.add_source(points_source)
         im = sc.render()
         im.write_png("points.png")
-        return im
```

Comment:
Hmm, I thought for a minute that removing the return would make this harder to debug, but now I'm not sure that it matters.  So I'm OK with this.